memsql_ops.service

```diff
@@ -1,10 +0,0 @@
-#!/bin/bash
-
-# ensure the volume is correct
-if [ ! -f /memsql/.setup ]; then
-    touch /memsql/.setup
-    cp -r /vol-template/* /memsql
```

Comment:
I think we should do the same here as with the port connections. The image now uses a standard volume setting and users that want to be able to have persistent data will be able to use the docker docs and the VOLUME line in the Dockerfile to do it.